crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownProceedingControllerTest.java

```diff
@@ -146,9 +146,9 @@ void givenAValidInput_whenUpdateIsInvoked_thenSuccess() throws Exception {
 
     @Test
     void givenAInValidInput_whenUpdateIsInvoked_RequestObjectFailsValidation() throws Exception {
-        var apiUpdateApplicationRequest =
```

Comment:
The other endpoints also have tests to check for a missing or empty request body, can we add these in please?